test/phpunit.xml

```diff
@@ -6,20 +6,13 @@
         <log type="coverage-php" target="build/logs/clover.serialized"/>
     </logging>
     <filter>
-        <whitelist processUncoveredFilesFromWhitelist="false">
-            <directory suffix=".php">../module/src</directory>
+        <whitelist>
+            <directory suffix=".php">../module/*/src</directory>
         </whitelist>
     </filter>
     <testsuites>
         <testsuite name="AllTests">
-       	    <directory>../module/Install/test/InstallTest</directory>
-            <directory>../module/Core/test/CoreTest</directory>
-            <directory>../module/Applications/test/ApplicationsTest</directory>
-            <directory>../module/Auth/test/AuthTest</directory>
-            <directory>../module/Auth/test/AclTest</directory>
-            <directory>../module/Jobs/test/JobsTest</directory>
-            <directory>../module/Organizations/test/OrganizationsTest</directory>
-            <directory>../module/Cv/test/CvTest</directory>
+            <directory>../module/*/test/*Test</directory>
```

Comment:
Hi Toni,
we currently cannot use wildcards here, because I eg. have also checked out the YawikXingVenderApi module in my modules directory. And these tests fail as long the YawikXingVendorApi module is not listed in the TestConfig.php